packages/terra-docs/README.md

```diff
@@ -0,0 +1,24 @@
+# Terra Docs
+
+[![NPM version](https://badgen.net/npm/v/terra-docs)](https://www.npmjs.com/package/terra-docs)
+[![Build Status](https://badgen.net/travis/cerner/terra-core)](https://travis-ci.com/cerner/terra-core)
+
+{insert description}
+
+- [Getting Started](#getting-started)
+- [Documentation](https://github.com/cerner/terra-core/tree/main/packages/terra-docs/docs)
```

Comment:
Starting a petition to redo all the branch renames `main` -> `base`... just kidding.

Reply:
![](https://media.giphy.com/media/12ZrvEf3lme15S/giphy.gif)